uniref-rest/src/main/java/org/uniprot/api/uniref/request/UniRefIdsSearchRequest.java

```diff
@@ -0,0 +1,81 @@
+package org.uniprot.api.uniref.request;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.PositiveOrZero;
+
+import lombok.Data;
+
+import org.springframework.http.MediaType;
+import org.uniprot.api.rest.request.IdsSearchRequest;
+import org.uniprot.api.rest.request.ReturnFieldMetaReaderImpl;
+import org.uniprot.api.rest.respository.facet.impl.UniRefFacetConfig;
+import org.uniprot.api.rest.validation.ValidContentTypes;
+import org.uniprot.api.rest.validation.ValidFacets;
+import org.uniprot.api.rest.validation.ValidReturnFields;
+import org.uniprot.api.rest.validation.ValidSolrQueryFacetFields;
+import org.uniprot.api.rest.validation.ValidSolrQuerySyntax;
+import org.uniprot.api.rest.validation.ValidUniqueIdList;
+import org.uniprot.store.config.UniProtDataType;
+
+import uk.ac.ebi.uniprot.openapi.extension.ModelFieldMeta;
+import io.swagger.v3.oas.annotations.Parameter;
+
+/**
+ * @author sahmad
+ * @created 18/03/2021
+ */
+@Data
+public class UniRefIdsSearchRequest implements IdsSearchRequest {
+    @NotNull(message = "{search.required}")
+    @Parameter(description = "Comma separated list of UniRef ids")
+    @ValidUniqueIdList(uniProtDataType = UniProtDataType.UNIREF)
+    private String ids;
+
+    @ModelFieldMeta(reader = ReturnFieldMetaReaderImpl.class, path = "uniref-return-fields.json")
+    @Parameter(description = "Comma separated list of fields to be returned in response")
+    @ValidReturnFields(uniProtDataType = UniProtDataType.UNIREF)
+    private String fields;
+
+    @Parameter(description = "Name of the facet search")
+    @ValidFacets(facetConfig = UniRefFacetConfig.class)
+    @ValidContentTypes(contentTypes = {MediaType.APPLICATION_JSON_VALUE})
+    private String facets;
+
+    @Parameter(
+            description =
+                    "Criteria to filter by facet value. It can any supported valid solr query.")
```

Comment:
any supported -> support any

find/replace solr for lucene (throughout in comments -- i think we said we'd do this once before for other descriptions, in a conv with leo)

Reply:
Thanks.. copy paste error :-)